terraform/environments/digital-prison-reporting/main.tf

```diff
@@ -668,6 +625,16 @@ module "glue_curated_zone_database" {
   aws_region     = local.account_region
 }
 
+# Glue Database Catalog for Prisons Fabric
+module "glue_prisons_database" {
```

Comment:
Creates a prisons database for the symlinked Hive tables which redshift will have a view of.